identify which manifest failed be loaded

With a malformed manifest file inside a chart, I saw this error when installing:

```
$ helm install deis-dev
[ERROR] Failed to load chart: error converting YAML to JSON: yaml: line 64: mapping values are not allowed in this context
```

Helm should identify which specific manifest failed to load so users have an easier time identifying which manifest they need to examine.

--- a/action/fetch.go
+++ b/action/fetch.go
@@ -55,7 +55,7 @@ func fetch(chartName, lname, homedir, chartpath string) {
 
 	fi, err := os.Stat(src)
 	if err != nil {
-		log.Warn("Oops. Looks like there was an issue finding the chart, %s, n %s. Running `helm update` to ensure you have the latest version of all Charts from Github...", lname, src)
+		log.Warn("Oops. Looks like there was an issue finding the chart, %s, in %s. Running `helm update` to ensure you have the latest version of all Charts from Github...", lname, src)
 		Update(homedir)
 		fi, err = os.Stat(src)
 		if err != nil {

--- a/manifest/manifest.go
+++ b/manifest/manifest.go
@@ -2,6 +2,7 @@
 package manifest
 
 import (
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
@@ -65,7 +66,7 @@ func Parse(filename string) ([]*Manifest, error) {
 
 	docs, err := codec.YAML.Decode(d).All()
 	if err != nil {
-		return ms, err
+		return ms, fmt.Errorf("%s %s", filename, err)
 	}
 
 	for _, doc := range docs {

